oap-server/server-starter/src/main/resources/ui-initialized-templates/os_windows/windows-service.json

```diff
@@ -125,6 +121,13 @@
           {
             "label": "available"
           }
+        ],
+        "metrics": [],
+        "metricMode": "Expression",
+        "typesOfMQE": [
+            "TIME_SERIES_VALUES",
+            "TIME_SERIES_VALUES",
+            "TIME_SERIES_VALUES"
```

Comment:
@lagagain Please use the latest UI to export, it is changed recently.

Reply:
Are there documents about ui-template settings?  Where I could found?

----------

I changed UI init setting with Download from Database after I manually changed the UI with version 9.7.
I can't understand what the setting means.

Reply:
This is UI settings, once you changed that, and Database is empty, they would be reimported.

And you cam change it from UI in local UI with edit mode, and export it to generate the JSON file.

It is changed in v10 dev about this JSON file, so, use the latest of the codes(not 9.7) and edit the changes.

Reply:
You should either use the latest UI or at least not change this part, and keep rename only.

Reply:
I remove the commit.